packages/example/pages/_app.tsx

```diff
@@ -6,8 +6,12 @@ import { AppProps } from 'next/app';
 import { Toaster } from 'react-hot-toast';
 
 function MyApp({ Component, pageProps, router }: AppProps): React.ReactElement {
-  if (router.route === '/wallet') {
-    return <Component {...pageProps} />;
+  if (router.route !== '/') {
```

Comment:
If I understand correctly it's because we don't want any page other than the index to be wrapped in the provider below.

Another solution that is more explicit (and easier to understand at a glance IMO) would be to move the `CeloProvider` to the page that uses it, `index.ts`. With that, we don't need any condition at the `_app.ts` level.

Reply:
Exactly @crondinini just a lot more changes to do that quick fix. Happy to open a refactor PR after v4 though.